docs/source/notebooks/Full_process_example_1.ipynb

```diff
@@ -11,9 +11,17 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/david/.conda/envs/birdy/lib/python3.7/site-packages/ipykernel_launcher.py:28: DeprecationWarning: xclim.subset is deprecated in xclim v0.19.1-beta. Please take note that xclim presently exposes the 'clisops' library subsetting API via `from clisops.core import subset`. This functionality may eventually change.\n"
```

Comment:
Should this be fixed?

Reply:
I think the warning is not necessary. To be fixed in xclim, yes I believe.

Reply:
We can slate this change for the next minor release.